src/main/java/com/fabriccommunity/spookytime/world/feature/SpookyOreFeature.java

```diff
@@ -0,0 +1,137 @@
+package com.fabriccommunity.spookytime.world.feature;
+
+import com.fabriccommunity.spookytime.registry.SpookyBlocks;
+import com.mojang.datafixers.Dynamic;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.BlockPos.Mutable;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.Heightmap.Type;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+import net.minecraft.world.gen.chunk.ChunkGeneratorConfig;
+import net.minecraft.world.gen.feature.Feature;
+
+import java.util.BitSet;
+import java.util.Random;
+import java.util.function.Function;
+
+public class SpookyOreFeature extends Feature<SpookyOreFeatureConfig> {
+    public SpookyOreFeature(Function<Dynamic<?>, ? extends SpookyOreFeatureConfig> function_1) {
+        super(function_1);
+    }
+
+    @Override
+    public boolean generate(IWorld iWorld_1, ChunkGenerator<? extends ChunkGeneratorConfig> chunkGenerator, Random random, BlockPos blockPos, SpookyOreFeatureConfig oreFeatureConfig) {
+        float float_1 = random.nextFloat() * 3.1415927F;
+        float float_2 = (float)oreFeatureConfig.size / 8.0F;
+        int int_1 = MathHelper.ceil(((float)oreFeatureConfig.size / 16.0F * 2.0F + 1.0F) / 2.0F);
+        double double_1 = (double)((float)blockPos.getX() + MathHelper.sin(float_1) * float_2);
+        double double_2 = (double)((float)blockPos.getX() - MathHelper.sin(float_1) * float_2);
+        double double_3 = (double)((float)blockPos.getZ() + MathHelper.cos(float_1) * float_2);
+        double double_4 = (double)((float)blockPos.getZ() - MathHelper.cos(float_1) * float_2);
+        double double_5 = (double)(blockPos.getY() + random.nextInt(3) - 2);
+        double double_6 = (double)(blockPos.getY() + random.nextInt(3) - 2);
+        int int_3 = blockPos.getX() - MathHelper.ceil(float_2) - int_1;
+        int int_4 = blockPos.getY() - 2 - int_1;
+        int int_5 = blockPos.getZ() - MathHelper.ceil(float_2) - int_1;
+        int int_6 = 2 * (MathHelper.ceil(float_2) + int_1);
+        int int_7 = 2 * (2 + int_1);
+
+        for(int int_8 = int_3; int_8 <= int_3 + int_6; ++int_8) {
+            for(int int_9 = int_5; int_9 <= int_5 + int_6; ++int_9) {
+                if (int_4 <= iWorld_1.getTop(Type.OCEAN_FLOOR_WG, int_8, int_9)) {
+                    return this.generateVeinPart(iWorld_1, random, oreFeatureConfig, double_1, double_2, double_3, double_4, double_5, double_6, int_3, int_4, int_5, int_6, int_7);
+                }
+            }
+        }
+
+        return false;
+    }
+
+    protected boolean generateVeinPart(IWorld world, Random random, SpookyOreFeatureConfig oreFeatureConfig, double double_1, double double_2, double double_3, double double_4, double double_5, double double_6, int int_1, int int_2, int int_3, int int_4, int int_5) {
+        int int_6 = 0;
```

Comment:
If I had to figure out what stuff did in the chunk generator pr to rename it you have to rename all these too

Reply:
You can just hardcode uses of int_6 to 0 if you don’t know what it does